src/tbots.py

```diff
@@ -0,0 +1,92 @@
+#!/opt/tbotspython/bin/python3.8
+
+import os
+import sys
+import iterfzf
+from subprocess import PIPE, run, check_call
+import argparse
+from thefuzz import fuzz
+from thefuzz import process
+
+# thefuzz is a fuzzy string matcher in python
+# https://github.com/seatgeek/thefuzz
+#
+# It returns a match ratio between the input and the choices
+# This is an experimentally determined threshold that works
+# for our bazel commands
+THEFUZZ_MATCH_RATIO_THRESHOLD = 50
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Run stuff")
+
+    parser.add_argument("action", choices=["build", "run", "test"])
+    parser.add_argument("search_query")
+    parser.add_argument("-v", "--enable_visualizer", action="store_true")
+    parser.add_argument("-s", "--stop_ai_on_start", action="store_true")
```

Comment:
it'd be nice if these weren't hard coded, i.e. what about passing in the network interface? What if we need to add new arguments in the future? If we had a `--args` argument to pass in a bunch of flags and then we had the C++ binary accept shortened flags, e.g. `-s`, then this script could be just pass the flags along to `test_args` or not depending on test vs run. Clearly, debug build, cache test results etc are good. So `tbots.py test cornerkick -d -p --args -s -v`

Reply:
network interface arg is supported, the arguments that aren't listed here are passed through, see `parse_known_args` docs

so the only special cases are the two short hand ones, its harder to get the C++ binary to accept shorthand because the dynamic param schema doesn't have a spot for it. We could make one but that seems overkill for just these 2 special cases

Reply:
I see, a comment would help

Reply:
added